### PR TITLE
xed: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/linux/xed.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`xed`
+
+- Open specific documents:
+
+`xed {{path/to/document{1,2,...}}}`
+
+- Open documents in a specific encoding:
+
+`xed --encoding {{WINDOWS-1252}} {{path/to/document{1,2,...}}}`
```
Comment (navarroaxel) [inline pages/linux/xed.md:16]:
```suggestion
`xed --encoding {{WINDOWS-1252}} {{path/to/document1 path/to/document2 ...}}`
```

Comment (EmilySeville7cfg) [reply pages/linux/xed.md]: Fixed in two examples at once 😄

**Review by blueskyson — APPROVED** [review]

File: pages/linux/xed.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open files in a specific encoding:
+
+`xed --encoding {{WINDOWS-1252}} {{path/to/file1 path/to/file2 ...}}`
+
+- Print all supported encodings:
+
+`xed --list-encodings`
+
+- Open file and go to a specific line:
```
Comment (mfrw) [inline pages/linux/xed.md:22]:
```suggestion
- Open a file and go to a specific line:
```

**Review by mfrw — APPROVED** [review]
LGTM with a teeny nitpick

File: pages/linux/xed.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start the editor:
+
+`xed`
+
+- Open specific files:
+
+`xed {{path/to/file1 path/to/file2 ...}}`
+
+- Open files in a specific encoding:
```
Comment (navarroaxel) [inline pages/linux/xed.md:14]:
```suggestion
- Open files using a specific encoding:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by mfrw** [comment]
@navarroaxel Ping :)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**